scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```diff
@@ -108,18 +109,22 @@ RUN set -ex; \
     # with sanitized version. Do not perform apt
     # remove because it removes mlton as well that
     # we need for building libabicoder
+    cd /usr/src/; \
+    git clone --depth 1 --branch gmp-6.2.1 https://github.com/gmp-mirror/gmp-6.2 gmp/; \
+    rm -r gmp/.git/; \
+    test \
+        "$(tar --create gmp/ --sort=name --mtime=1970-01-01Z --owner=0 --group=0 --numeric-owner | sha256sum)" = \
+        "d606ff6a4ce98692f9920031e85ea8fcf4a65ce1426f6f0048b8794aefed174b  -"; \
+    # NOTE: This removes also libgmp.so, which git depends on
```

Comment:
And also `wget`, and probably a ton of other stuff.

Reply:
Yeah, but git is relevant because we're using it a few lines above. The comment is there to explain why the removal of the installed GMP has to be done here and not before.